pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -311,6 +311,14 @@ spec:
               # OPENEBS_IO_CSTOR_ID env has UID of cStorPool CR.
             - name: OPENEBS_IO_CSTOR_ID
               value: {{ pluck .ListItems.currentRepeatResource .ListItems.nodeUidMap.nodeUid |first | splitList " " | first}}
+            - name: OPENEBS_POOL_POD_NAME
```

Comment:
`POD_NAME` vs `OPENEBS_POOL_POD_NAME` and `OPENEBS_POOL_POD_NAMESPACE` vs `NAMESPACE` ?

Reply:
POD_NAME & POD_NAMESPACE will suffice here. Openebs need not be prefixed unless and until kubernetes sets these environment variables itself.

Does kubernetes sets some env variables for pod name & namespace by any chance?

Reply:
Ok. I will do it.